package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@segment/integration-intercom",
-  "version": "2.5.0",
+  "version": "2.6.0",
```

Comment:
(but strict semver doesn't work for anyone, so whatever you're most comfortable with)